README.md

```diff
@@ -1,7 +1,7 @@
 <a name="readme-top"></a>
 
 <div align="center">
-  <h3><b>README Template</b> <h3>
+  <h3><b>Microverse README Template</b></h3>
```

Comment:
- [ OPTIONAL ] Please make sure to remove this line as code as it is irrelevant in your README file :+1: